dev/WindowsAppSDK_Insights/WindowsAppSDKInsightsLib/WindowsAppSDKInsightsLib.sln

```diff
@@ -0,0 +1,37 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
This solution file is not needed. You should be able to add a reference to the vcxproj in your solution. Remove.

Reply:
When I use origin/main as-is and try building the Insights project in VS, I get this:

```
Severity	Code	Description	Project	File	Line	Suppression State

Error		This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is ..\packages\Microsoft.Windows.ImplementationLibrary.1.0.210803.1\build\native\Microsoft.Windows.ImplementationLibrary.targets.	S:\temp\WindowsAppSDK\dev\WindowsAppSDK_Insights\WindowsAppSDKInsightsLib\WindowsAppSDKInsightsLib.vcxproj	S:\temp\WindowsAppSDK\dev\WindowsAppSDK_Insights\WindowsAppSDKInsightsLib\WindowsAppSDKInsightsLib.vcxproj	185	

Error		Solution is not saved. Please save your solution before managing NuGet packages.				
```

Reply:
@ZenBird, given the reason above, are you OK with keeping the solution file?

Reply:
moot after https://github.com/microsoft/WindowsAppSDK/pull/1268